grails-app/conf/UrlMappings.groovy

```diff
@@ -1,10 +1,9 @@
-import grails.util.Environment
-
 class UrlMappings {
 
     static mappings = {
 
         "/proxy"(controller: "proxy", action: "index")
+        "/sitemap/**"(controller: "sitemap", action: "index")
 
```

Comment:
What your talking about was going to be changed in https://github.com/aodn/aodn-portal/pull/2407. But its merged!
I'll add a seperate pull request if a transparent redirect can be acheived